pkg/cmd/estimate.go

```diff
@@ -13,6 +13,9 @@ import (
 
 var e_url string
 var e_apiKey string
+var e_cache_forecast bool
+var e_cache_file_prefix string
+var e_cache_time int32
```

Comment:
in this file modifing the `CheckEstimate` function please add a check to verify `e_cache_time` is >=0